lib/sierra-models/base.js

```diff
@@ -71,13 +71,7 @@ class SierraBase {
     }
   }
 
-  // This function takes a marc tag, and returns an array of primary values with
-  //   relevant parallel values attached and or orphaned parallels relevant to
-  //   given marc tag
-  _fieldsWithParallelsAndOrPrimaries (marc) {
-    // _varFieldByMarcTag returns array of raw marc fields filtered by marc tag:
-    //  [{fieldTag: 'a', marcTag: '123', content: 'the thing'}]
-    const primaryRawFields = this._varFieldByMarcTag(marc)
+  _buildParallelsLookup (marc) {
```

Comment:
this was extracted for an approach I did not end up using, but I neglected to reintegrate it.